lib/committee/bin/committee_stub.rb

```diff
@@ -8,6 +10,8 @@ class CommitteeStub
       def get_app(schema, options)
         cache = {}
 
+        raise Committee::NotSupportOpenAPI3.new("OpenAPI3 not support stub") unless schema.support_stub?
```

Comment:
Could you rename:

* `NotSupportOpenAPI3` -> `OpenAPI3Unsupported`
* "OpenAPI3 not support stub" -> "Stubs are not yet supported for OpenAPI 3"